setup.py

```diff
@@ -110,6 +113,7 @@ def parse(root):
         "Topic :: System :: Software Distribution",
         "Topic :: Utilities",
     ],
+    install_requires=["toml"],
```

Comment:
This should use an extra for now

Reply:
I just noted the last comment which means just that

Reply:
Done in 6fb77d9